### PR TITLE
raise max value of view_pitch dvars and makes the game use them

### PR DESCRIPTION
**Pull request by panelafoda**
raise max value of view_pitch dvars and makes the game use them

### CODE REVIEW

**Comment by diamante0018** [comment]
You need to look at your code in "Files Changed". It's completely formatted the wrong way.

Also, it would be a good idea to register a new dvar so users can turn off this feature.

**Comment by diamante0018** [comment]
snake_case your code as well. The entire project uses snake_case

**Comment by panelafoda** [comment]
done. sorry its my first time using github pull requests

File: src/client/component/backflips.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#include <std_include.hpp>
+#include "loader/component_loader.hpp"
+#include "game/game.hpp"
+
+#include <utils/hook.hpp>
+#include "dvars.hpp"
+
+namespace backflips
+{
+	//returning 1 makes the game use the dvar value instead of hardcoded value
```
Comment (mjkzy) [inline src/client/component/backflips.cpp:10]: add this comment above all the `utils::hook:call` instead maybe?

File: src/client/component/backflips.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	            	utils::hook::call(0x6E6823_b, use_view_pitch_dvars);
+	
+	            	utils::hook::call(0x5B5BBB_b, use_view_pitch_dvars);
+	            	utils::hook::call(0x5B5BE0_b, use_view_pitch_dvars);
+	
+	            	utils::hook::call(0x154E81_b, use_view_pitch_dvars);
+	            	utils::hook::call(0x154EA1_b, use_view_pitch_dvars);
+	
+	            	utils::hook::call(0x5651E_b, use_view_pitch_dvars);
+	            	utils::hook::call(0x56555_b, use_view_pitch_dvars);
```
Comment (mjkzy) [inline src/client/component/backflips.cpp:35]: remove spacing in between if it's all one function

**Review by mjkzy — APPROVED** [review]
nit picking in the comments, but PR looks good

**Pull request merged**